operator/apis/options/v1alpha1/keptnconfig_types.go

```diff
@@ -29,11 +29,13 @@ type KeptnConfigSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// OTelCollectorUrl can be used to set the Open Telemetry collector that the operator should use
+	// +optional
 	OTelCollectorUrl string `json:"OTelCollectorUrl,omitempty"`
 	// KeptnAppCreationRequestTimeout is used to set the interval in which automatic app discovery
 	// searches for workload to put into the same auto-generated KeptnApp
 	// +kubebuilder:default:=30
-	KeptnAppCreationRequestTimeout uint `json:"keptnAppCreationRequestTimeout,omitempty"`
+	// +optional
+	KeptnAppCreationRequestTimeoutSeconds uint `json:"keptnAppCreationRequestTimeoutSeconds,omitempty"`
```

Comment:
pls also change the docs string when you change the variable name here.

Reply:
do you mean something different than this?
https://github.com/keptn/lifecycle-toolkit/pull/1097/files#diff-7ab58e37e6f4412bbcc3d577c257684bae6e4bd7629ea0b57d38d996ad5119fcL66

Reply:
gotcha: https://github.com/keptn/lifecycle-toolkit/pull/1126